### ISSUE TITLE
A&R -> variance report and review screen improvements

### ISSUE THREAD

**Issue by meyerdarcie** [bcgov/bcparks-ar-admin#354]
#### Description:

Once you review a piece data from the Review tab, it takes you out of the Review tab. The only way back is to use the browser’s back button, otherwise you need to reselect Review Data and then re-search your variances.  Thinking it needs some sort of “Return to Review Data Search” or similar button. Screen Grab 1: example of variance search result. Clicking on View/Edit opens the data entry screen.

![Image](https://github.com/user-attachments/assets/27a8b62a-9dea-4221-9adc-e994baf3aaff)

#### Acceptance Criteria:
[Note: Use 'Given/When/Then' format if it makes sense to. Otherwise, a simple checklist that can be tested.]

#### Development Checklist:

- [ ] ...
- [ ] ...
- [ ] ...

#### Dependencies

- Blocked by
- Blocking

**Relevant documentation as reference**

**Definition of Ready**

- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [ ] Design / Solution is accepted by Product Owner (if applicable)
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- 
-

**Comment by LindsayMacfarlane** [comment]
Backlog refinement (October 8): This ticket to be implemented for spring season.

**Comment by LindsayMacfarlane** [comment]
May 14th A&R check-in with PROS team. This was flagged as a med-lower priority item to solve. The team indicated that as it is currently set up, it isn't functional. They have work arounds in the interim.
